src/archivematicaCommon/lib/storageService.py

```diff
@@ -200,7 +200,7 @@ def browse_location(uuid, path):
     return browse
 
 
-def wait_for_async(response, poll_seconds=2):
+def wait_for_async(response, poll_seconds=2, poll_timeout=600):
     """
     Poll for results on an async endpoint.  `response` should have a HTTP 202
     (Accepted) status code, and is expected to contain a Location header telling
```

Comment:
Unless a reader looks back over the issue then understanding the poll_timeout is in seconds will take you back to the request lib docs to ensure that 600 is 10 mins, and not say, 0.6ms (I believe anyway, I couldn't see it anywhere so just followed that process). While it is unlikely to be the latter! That being said, can it be made explicit in the variable name, or the docstring what the defaults are in this case?

Reply:
Thanks @ross-spencer, good thought.  I've done both: renamed the variable to make the units obvious, and added a description of the two optional parameters to the docstring.